puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/tests/testForeman.groovy

```diff
@@ -0,0 +1,135 @@
+node('fast') {
+
+    stage('Setup Git Repos') {
+
+        git url: "https://github.com/theforeman/foreman", branch: "develop"
+
+    }
+
+    stage("Setup RVM") {
+
+        withRVM([
+            'rvm gemset empty --force',
+            'gem update --no-ri --no-rdoc',
+            'gem install bundler --no-ri --no-rdoc'
+        ])
+
+    }
+
+    stage('Configure Environment') {
+
+        try {
+            sh "cp config/settings.yaml.example config/settings.yaml"
+            sh "sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.yaml"
+            sh "sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml"
+
+            sh "cp $HOME/postgresql.db.yaml config/database.yml"
```

Comment:
Yes and no. This technically comes from -- https://github.com/theforeman/foreman-infra/blob/2b425b40fca1f790f0b25ae7a8811485d26f11c0/puppet/modules/slave/files/db/postgresql which means its managed by this repo.